src/templates/en/2019/chapter_prevnext.html

```diff
@@ -0,0 +1,25 @@
+<nav id="chapter-navigation">
```

Comment:
I don't think we need to generate these as part of the markdown rendering. We have the chapter info in `2019.json` and we can render them server-side.

Also we might not launch with all 20 chapters, so some are marked `todo` in the config. These links should go to the next/previous sequential chapter that isn't still marked "todo". That should be done dynamically on the server-side because it could change after the markdown is initially rendered.